src/net/tcp/listener.rs

```diff
@@ -41,6 +44,20 @@ pub struct TcpListener {
 }
 
 impl TcpListener {
+    /// Creates a new `TcpListener` from a standard `net::TcpListener`.
+    ///
+    /// This function is intended to be used to wrap a TCP listener from the
+    /// standard library in the Mio equivalent. The conversion assumes nothing
+    /// about the underlying listener.
```

Comment:
Maybe add a note saying that the listener is not put in non-blocking mode as that is up to the user.

Reply:
Same for the other types.